CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ Security - in case of vulnerabilities.
 
 ## [Unreleased]
 
+## 2.0.1 (2024-02-29)
```

Comment:
Maybe it should release it as `2.1.0`

Reply:
I doubted about that as well, I changed it to `2.1.0`